### DISCUSSION TITLE
Was the addition of bitwardenAttachmentByRef necessary?

### DISCUSSION THREAD

**Discussion by josevh** [twpayne/chezmoi#2793] [Q&A]
I saw the latest release notes and the addition of `bitwardenAttachmentByRef` (from #2765) but I am not sure it was necessary.

The following templates return the same values for me:
```
chezmoi execute-template '{{ (bitwardenAttachment "id_ed25519.pub" "my-item-name") }}'
```
```
chezmoi execute-template '{{ (bitwardenAttachmentByRef "id_ed25519.pub" "item" "my-item-name") }}'
```

The second arg of `bitwardenAttachment` already worked with an item name. Am I missing something?

**Comment by josevh** [comment]
The following also returns the same values:
```
bw get attachment "id_ed25519.pub" --itemid "{ITEM_ID}"
```
```
bw get attachment "id_ed25519.pub" --itemid "{ITEM_NAME}"
```

I don't see the CLI docs implying that an item name would also be acceptable.
I suppose adding a method to use _documented_ functionality is a good thing.

**Comment by twpayne** [comment]
cc @cawaltrip
